chia/full_node/full_node.py

```diff
@@ -1547,7 +1545,6 @@ async def prevalidate_blocks(
         self,
         blockchain: AugmentedBlockchain,
         blocks_to_validate: list[FullBlock],
-        peer_info: PeerInfo,
```

Comment:
I take it `peer_info` was unused?
There's a comment below that still talks about `peer_info`. it should probably be removed as well